src/routes/(console)/project-[project]/overview/platforms/wizard/android/step3.svelte

```diff
@@ -3,15 +3,20 @@
     import Id from '$lib/components/id.svelte';
     import { WizardStep } from '$lib/layout';
     import { sdk } from '$lib/stores/sdk';
+    import { isSelfHosted } from '$lib/system';
 
     const { endpoint, project } = sdk.forProject.client.config;
     const code = `import io.appwrite.Client
 import io.appwrite.services.Account
 
-val client = Client(context)
+${
+    !isSelfHosted
+        ? `val client = Client(context).setProject("${project}")`
+        : `val client = Client(context)
```

Comment:
Can we flip this? So 
```
isSelfHosted
        ? `val client = Client(context)
        : `val client = Client(context).setProject("${project}")
```

Reply:
Doesn't seem correct.
We need to show endpoint and self-signed only on self-hosted instances.
What we could do is use `isCloud` instead of `!isSelfHosted`, that works?

Reply:
I just changed to using `isCloud` so for better readability. Also patched up the `react-native` sdk info.